cmake/Install.cmake

```diff
@@ -243,6 +243,8 @@ endif()
 
 if( APPLE )
   set(CPACK_PACKAGE_DEFAULT_LOCATION "/Applications")
+  set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
```

Comment:
The Qt based Installer Framework (IFW) has a regex for the version string.  It requires only numerics.  Earlier in this file, we give CPack a version identifier that is like 8.9.0-abcd123456, and it will fail for the alpha characters in the SHA.  This changes the version string to just not have the SHA on it -- only for Mac installers -- and allows the installer to be built.